middle-tier/src/main/java/com/marklogic/grove/boot/os/OSController.java

```diff
@@ -0,0 +1,4 @@
+package com.marklogic.grove.boot.os;
+
+public class OSController {
```

Comment:
@dbaldry did you mean to add this file?

Reply:
No - that shouldn't have been added! I've deleted it from the branch